docs/src/local_optimizers/local_derivative_free.md

```diff
@@ -68,3 +68,27 @@ for more information on the algorithms. Possible algorithm names are:
 * `LN_SBPLX`
 * `LD_MMA`
 * `LD_CCSAQ`
+
+The following optimizer parameters can be set as `kwargs`:
```

Comment:
Yeah... I really am thinking about whether we should be changing the docs to be per wrapper library instead.

Reply:
I agree that would make sense. I am writing another PR at the moment which could potentially include that.